Modules/TRD/src/PulseHeightTrackMatch.cxx

```diff
@@ -59,35 +47,15 @@ void PulseHeightTrackMatch::retrieveCCDBSettings()
 
 void PulseHeightTrackMatch::buildHistograms()
 {
-  mParsingTimePerTF.reset(new TH1F("parsingtimeperTF", "Time for processing each TF", 100000, 0, 100000));
-  mParsingTimePerTF->GetXaxis()->SetTitle("Time in #mus");
-  mParsingTimePerTF->GetYaxis()->SetTitle("Counts");
-  getObjectsManager()->startPublishing(mParsingTimePerTF.get());
-
-  mTracksPerEvent.reset(new TH1F("tracksperevent", "Matched TRD tracks per Event", 100, 0, 10));
-  getObjectsManager()->startPublishing(mTracksPerEvent.get());
-
-  mTrackletsPerMatchedTrack.reset(new TH1F("trackletspermatchedtrack", "Tracklets per matched TRD track", 100, 0, 10));
-  getObjectsManager()->startPublishing(mTrackletsPerMatchedTrack.get());
-
-  mPulseHeightpro.reset(new TProfile("PulseHeight/mPulseHeightpro", "Pulse Height profile plot;Timebins;ADC Counts", 30, -0.5, 29.5));
+  mPulseHeightpro.reset(new TProfile("PulseHeight/mPulseHeightpro", "PulseHeight;Timebins;ADC Counts", 30, -0.5, 29.5));
   drawLinesOnPulseHeight(mPulseHeightpro.get());
   mPulseHeightpro.get()->Sumw2();
   getObjectsManager()->startPublishing(mPulseHeightpro.get());
 
-  mPulseHeightperchamber.reset(new TProfile2D("PulseHeight/mPulseHeightperchamber", "mPulseHeightperchamber;Timebin;Chamber", 30, -0.5, 29.5, 540, 0, 540));
+  mPulseHeightperchamber.reset(new TProfile2D("PulseHeight/mPulseHeightperchamber", "PulseHeight per chamber;Timebin;Chamber", 30, -0.5, 29.5, 540, 0, 540));
```

Comment:
what does this plot actually tell us?

Reply:
i think, its the same mean of ADC per chamber.

Reply:
OK, and what can we learn from that plot? I have seen it once in a meeting I think and to me it is not clear what we should be able to see there